BEGINNER_GUIDE.md

```diff
@@ -0,0 +1,167 @@
+```
+git clone https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME
+```
+*where YOUR_GITHUB_USER_NAME is your Github user name and PROJECT_REPOSITORY_NAME is the name of the project repository.*
+
+
+**Step 3: Remote**
+
+First, navigate to your local repository using:
+```
+cd PROJECT_REPOSITORY_NAME
+``` 
+
+The default remote name of the local repository is *origin*. This remote points to the forked repository. But you need to keep track of the main project repository. For this, we need to create a new remote named *upstream*. 
+
+Run the following command to add new remote:
+```
+git remote add upstream https://github.com/anitab-org/PROJECT_REPOSITORY_NAME
+```
+
+Use `git remote -v` to check the current remote. It should show the following output:
+```
+$ git remote -v
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (fetch)
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (push)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (fetch)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (push)
+``` 
+
+
+**Step 4: Pull latest changes from upstream**
+
+It is pretty obvious that their might be some changes in the main project after you have forked it. Your local repository should be synchronous to the main repository.
+
+For this, you have to pull changes from upstream master branch to your local repository.
+
+Run the following command to pull the latest changes:
+```
+git pull upstream master
+```
+
+
+**Step 5: Creating new branch**
+
+It is a good practice to a create separate branch for every issue so that it is isolated from the *master* branch.
+
+Use the following command to create a new branch and switch to it:
+```
+git checkout -b BRANCH_NAME
+```
+
+*Try to choose appropriate BRANCH_NAME. I would recommend using issue number as branch name (as it is unique to every issue).*
+
+
+**Step 6: Solving the issue**
+
+Now you can start working on the issue. You may need to read the installation guide that is specific to every project. For [VMS](https://github.com/anitab-org/vms) project you can check it's [installation guide](https://github.com/anitab-org/vms/blob/develop/aut_docs/Installation_Setup.md).
+
+Make the relevant changes that can fix the issue.
+
+***Stuck?***
+
+Don't worry! We are always here to help you. Go to the comment section of the issue and ask your doubt. Someone will help you out. You can also join our [ AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/) channel to discuss any topic.
+
+After approval, you must make continuous notes on your progress in the issue while working. If there is not at least one comment every 3 days, the maintainer can reassign the issue.
+
+
+**Step 7: Commit changes**
+
+*Repeating this step after every significant change in your local repository is a good practice.*
+
+Use ` git add -A ` to add all files, except the files that are mentioned in .gitignore.
```

Comment:
Dont use git add -A line it will also add files like newly created virtual env etc
Update to git add file_name where file_name is the file in which you have created/updated changes
Please look at the syntax once on google